streaming/base/shuffle/py1b.py

```diff
@@ -34,6 +34,7 @@ def get_shuffle_py1b(shard_sizes: NDArray[np.int64],
     Returns:
         NDArray[np.int64]: 1:1 mapping of sample ID to shuffled sample ID.
     """
+    block_size = int(block_size)
```

Comment:
Why this ?

Reply:
Ah I see. I feel it should be fixed in the llm-foundry since they are passing it as float. I would recommend to remove it from here.